python/pyspark/streaming/kafka.py

```diff
@@ -70,7 +71,103 @@ def createStream(ssc, zkQuorum, groupId, topics, kafkaParams={},
         except Py4JJavaError, e:
             # TODO: use --jar once it also work on driver
             if 'ClassNotFoundException' in str(e.java_exception):
-                print """
+                KafkaUtils._printErrorMsg(ssc.sparkContext)
+            raise e
+        ser = PairDeserializer(NoOpSerializer(), NoOpSerializer())
+        stream = DStream(jstream, ssc, ser)
+        return stream.map(lambda (k, v): (keyDecoder(k), valueDecoder(v)))
+
+    @staticmethod
+    def createDirectStream(ssc, brokerList, topics, kafkaParams={},
```

Comment:
OK, that make sense, I will ignore this handler support in Python API unless someone has specific requirement for this.